docs/api/transactions/create.md

```diff
@@ -1,7 +1,93 @@
 ---
 id: create
-title: Create
+title: Create Transaction
 sidebar_label: Create
+keywords:
+  - OpenHIM
+  - API
+  - Transaction
+  - Routing
+  - Create
+description: Create an OpenHIM Transaction
 ---
 
-Coming soon ...
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Create Transaction via the server
+
+The OpenHIM routes through the ports `5000 (https)`, `5001 (http)` and `7786 tcp`. Any request that passes through the OpenHIM is a transaction. For the routing to happen, there has to be a channel that has the route(s) to the upstream server(s). The request has to have a path that matches the channel's `urlPattern`.
+
+To route using the OpenHIM, create a channel on the OpenHIM console or through the [API](../channels/create). If the channel is configured to be a private channel, a client will need to be [created](../clients/create). The client and the channel should be configured such that they are linked. This is done by ensuring that one of the `roles` created in client matches one of the `roles` in channel's `allow` property. The client id and password will be used for authentication. The transaction is created irregardless of whether the upstream server (mediator) being routed to exists or not.
+
+## Example routing
+
```

Comment:
@bradsawadye As part of this example I think it would look better to point to a real endpoint - ie: the openAPI Martin suggested - instead of letting the transaction fail.
The failing transaction just looks bad.

Reply:
The endpoint i ended up using was to pull the FHIR Patient Example JSON object
https://www.hl7.org/fhir/patient-example.json

Just to make it a bit more real world

Reply:
@MattyJ007, I get what you are saying but the user that runs the script has to create a channel first and configure it. The destination of the request is up to them (as martin said in the comment above). The transaction will fail if the channel is not configured right, and when the urlPattern does not match the path.

Reply:
@bradsawadye I think my comment might have been a bit misleading. We do not want to force the user onto a specific endpoint we defined (As was done with the integrated server we were routing to), but instead guide them to a working solution but one that they can choose to change if they wish.

If you have a look at the getting started section, ive added an example section that informs the user on what fields to supply for channel/client creation, but also a import.json file for ease of use. Within these steps i define the endpoint we will be reaching but the user can still decide to change it if they want